shoreline/README.md

```diff
@@ -48,7 +48,7 @@ For example:
 
 For in-depth instructions on configuring the app, see the [Datadog-Shoreline documentation][4].
 
-## Troubleshooting
+## Support
```

Comment:
Note: for manifest v2 apps, this header needs to be `Support` instead of `Troubleshooting`.